Systems/c172p-engine.xml

```diff
@@ -62,7 +62,100 @@
             </function>
             <output>/fdm/jsbsim/propulsion/engine[1]/air-intake-impedance-factor</output>
         </fcs_function>
-        
+
+    </channel>
+
+    <channel name="damage">
+        <fcs_function name="/damage/rpm-over-redline">
```

Comment:
Path should be relative. Since this function is in the `c172p-engine.xml` file, you might as well change the attribute to `name="engine/rpm-over-redline"`. (I just mentioned `damage/` and `heat/` as an example)